examples/amp-consent/diy-3p-iframe-tcf-postmessage.html

```diff
@@ -211,6 +211,25 @@
         // should get response from window.top's CMP
         console.log('Minimal TCData return:', tcData);
       });
+      
+      const callback = (tcData, success) => {
+        console.log('addEventListenerCallback:', tcData)
```

Comment:
remove these `console.log`s?

Reply:
I'm leaving them here for developer debugging as this page is meant as an example :)